renderer/components/Onboarding/Steps/Recover.js

```diff
@@ -82,6 +258,30 @@ class Recover extends React.Component {
     wizardState: {},
   }
 
+  /**
+   * Create a mapping of first letter to words in BIP 39 list so
+   * we don't have to filter through the whole array when looking up values
+   * every time
+   */
+  bip39WordMap = bip39.reduce((bip39WordMap, word) => {
+    var firstLetter = word[0]
+    if (!bip39WordMap[firstLetter]) {
+      bip39WordMap[firstLetter] = []
+    }
+
+    bip39WordMap[firstLetter].push(word)
+
+    return bip39WordMap
+  }, {})
+
+  constructor(props) {
```

Comment:
@korhaliv Looking at this I realized there is no need to pass the seed as a prop to `Recover` component. This seed is never used for anything useful since you can't type out part of the seed, proceed further and return back to the recover form to see your partially typed out seed. You can click on the `Start over` button but this starts the process all over again with an empty seed.

Is it OK to remove this prop completely then?

Reply:
Also, you are right about the `state` variable. None of the state variables are actually needed and just `formState` is sufficient. I will remove them. If there is a good reason to have these state variables they can always be brought back.

Reply:
Done.